aospy/data_loader.py

```diff
@@ -151,8 +152,9 @@ def _prep_time_data(ds):
     """
     ds = times.ensure_time_as_dim(ds)
     ds, min_year, max_year = times.numpy_datetime_workaround_encode_cf(ds)
-    if internal_names.TIME_BOUNDS_STR in ds:
+    if TIME_BOUNDS_STR in ds:
         ds = times.ensure_time_avg_has_cf_metadata(ds)
+        ds[TIME_STR].values = times.average_time_bounds(ds).values
```

Comment:
If you went with my suggestion for the simpler method below, I *think* you could simplify this to:
```python
ds[TIME_STR] = times.average_time_bounds(ds)
```